README.md

```diff
@@ -152,7 +152,7 @@ build:
 
 ```toml
 [dependencies]
-chrono-tz = { version = "0.6", features = [ "filter-by-regex" ] }
+chrono-tz = { version = "0.9", features = [ "filter-by-regex" ] }
```

Comment:
Let's just ditch this bit of README, I think everyone knows how to do this now.

Reply:
Done.